tools/gfx/vulkan/render-vk.cpp

```diff
@@ -1033,6 +1033,8 @@ VkPipelineShaderStageCreateInfo VKRenderer::compileEntryPoint(
 
 SlangResult VKRenderer::initialize(const Desc& desc, void* inWindowHandle)
 {
+    SLANG_RETURN_ON_FAIL(slangContext.initialize(desc.slang, SLANG_SPIRV, "sm_5_1"));
```

Comment:
It seems a bit odd to use `sm_5_1` as the profile here. Surely it should be `glsl_440` or whatever is the appropriate GLSL version for what we need/want to support?